root/user/UserProfile.js

```diff
@@ -505,87 +505,101 @@ const UserProfileStatistics = ({
         <thead>
           <tr>
             <th colSpan="2">
-              {exp.l(
+              {$c.user ? exp.l(
                 'Edits ({view_url|view})',
                 {view_url: `/user/${encodedName}/edits`},
-              )}
+              ) : l('Edits')}
             </th>
           </tr>
         </thead>
 
         <tbody>
           <UserProfileProperty name={lp('Accepted', 'edit descriptor')}>
-            {exp.l(
+            {$c.user ? exp.l(
               '{count} ({view_url|view})',
               {
                 count: formatCount($c, editStats.accepted_count),
                 view_url: `/user/${encodedName}/edits/accepted`,
               },
-            )}
+            ) : (exp.l('{count}', {
```

Comment:
Is that string useful? I figured `formatCount` we be enough to localize the number.

Reply:
... that's a fair point :) Will amend.